### PR TITLE
fix: #66 and improved signup UI

### PR DESCRIPTION
**Pull request by kartikpandey2**
Explain the **details** for making this change. What existing problem does the pull request solve?

Username/Email field is hiding behind the navbar. #66

### CODE REVIEW

**Comment by kartikpandey2** [comment]
screenshot 155

File: fact-bounty-client/src/pages/main/components/MainLayout.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ class MainLayout extends Component {
 
 	render() {
 		return (
-			<div className="mainLayout">
+			<div className="mainLayout" style={{paddingTop: '70px'}}>
```
Comment (root5533) [inline fact-bounty-client/src/pages/main/components/MainLayout.js]: Remove inline css and take the padding style to mainLayout class

Comment (kartikpandey2) [reply fact-bounty-client/src/pages/main/components/MainLayout.js]: @root5533  done, please review 😃

**Comment by root5533** [comment]
This issue was address in pull request #87.

**Pull request closed without merging**